app/javascript/components/Footer/Footer.tsx

```diff
@@ -1,12 +1,64 @@
-import * as React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Can we take out `useState` and `useEffect`? I don't see them being used anywhere in this file.